index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#">
+                <img class="logo" src="/images/adesoji-dara-logo.png" alt="logo-image">
```

Comment:
The main goal of this project is to implement the given design pixel perfectly. Because that's what you will see on the jobs ads. Even though your logo looks cool 😎 , please stick to the design, remove the logo and add an `<a>` element with John Doe as logo like on the Figma template.